src/Application/Commands/ArchiveShoppingList/ArchiveShoppingListModel.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lindyhopchris\ShoppingList\Application\Commands\ArchiveShoppingList;
+
+class ArchiveShoppingListModel
```

Comment:
Required: good work with this class. At this point, this is testable. So as part of this PR I'd expect to see a test for this class added.